.checkstyle

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<fileset-config file-format-version="1.2.0" simple-config="false" sync-formatter="false">
+  <local-check-config name="maven-checkstyle-plugin checkstyle" location="file:/Users/lucas/git/Open-Realms-of-Stars/buildtools/checkstyle/ruleset.xml" type="remote" description="maven-checkstyle-plugin configuration checkstyle">
```

Comment:
Problem is that it contains absolute path to checkstyle ruleset:
location="file:/Users/lucas/git/Open-Realms-of-Stars/buildtools/checkstyle/ruleset.xml"

Can this be relative? If not then this configuration file needs to be added to gitignore file and not
added into repository.